apps/indexer/lib/indexer/block/fetcher.ex

```diff
@@ -182,20 +189,26 @@ defmodule Indexer.Block.Fetcher do
          logs = tx_logs ++ process_extra_logs(extra_logs),
          transactions_with_receipts = Receipts.put(transactions_params_without_receipts, receipts),
          %{token_transfers: normal_token_transfers, tokens: normal_tokens} = TokenTransfers.parse(logs),
-         gold_token_enabled = config(:enable_gold_token),
-         {:ok, gold_token, stable_token, oracle_address} <-
-           (if gold_token_enabled do
+         celo_token_enabled = config(:enable_gold_token),
+         {:ok,
+          %{
+            celo: celo_token,
+            cusd: stable_token_usd,
+            ceur: _
```

Comment:
missing something? or is `_` it?

Reply:
It means that we don't care about getting the `ceur` address in this case but we need both `celo` and `cusd`. It's like skipping certain object fields during the destructuring.